app/styles/charts.scss

```diff
@@ -103,9 +104,10 @@
   margin: auto;
 }
 
-.chart-card-title {
+h3.chart-card-title {
```

Comment:
i'll merge it to get it in but generally i think putting html elements in css selectors makes them too difficult to re-use elsewhere